docs/current/stubs.md

```diff
@@ -69,13 +69,10 @@ Creates an anonymous stub function
 
 #### `var stub = sinon.stub(object, \"method\");`
 
-Replaces `object.method` with a stub function.
+Replaces `object.method` with a stub function. An exception is thrown if the property is not already a function, to help avoid typos when stubbing methods.
```

Comment:
I would shorten the addition to "An exception is thrown if the property is not already a function."

We generally try to steer clear of explaining motivation, and only document behaviour.